docs/connector-development/config-based/understanding-the-yaml-file/declarative_oauth_examples/refresh_token.yml

```diff
@@ -0,0 +1,185 @@
+version: 6.13.0
+
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  stream_names:
+    - moves
+
+definitions:
+  streams:
+    moves:
+      type: DeclarativeStream
+      name: moves
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /api/v2/move/
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestPath
+          pagination_strategy:
+            type: CursorPagination
+            cursor_value: "{{ response.get('next') }}"
+            stop_condition: "{{ response.get('next') is none }}"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/moves"
+  base_requester:
+    type: HttpRequester
+    url_base: https://pokeapi.co
+    authenticator:
+      type: OAuthAuthenticator
+      refresh_request_body: {}
+      grant_type: refresh_token
+      client_id: '{{ config["client_id"] }}'
+      client_secret: '{{ config["client_secret"] }}'
+      refresh_token: '{{ config["client_refresh_token"] }}'
+      access_token_value: '{{ config["client_access_token"] }}'
+      access_token_name: access_token
+      refresh_token_updater:
+        refresh_token_name: refresh_token
+        refresh_token_config_path:
+          - client_refresh_token
+      token_refresh_endpoint: >-
+        https://yourconnectorservice.com/oauth/refresh?client_id={{client_id_value}}&client_secret={{client_secret_value}}&&refresh_token={{refresh_token}}
```

Comment:
This is incorrectly provided. The placeholders are not going to be resolved as expected, because the `token_refresh_endpoint` is the CDK owned property and could interpolate the `config[*]` values only. 

It's not expected this string to be used within the OAuthFlow, but to `refresh` the expired `access_token` only.

The closest example here is: `https://yourconnectorservice.com/oauth/refresh` with no query params. The params are added in the `refresh_token_request_body` and constructed at the CDK level.

Reply:
@bazarnov makes sense! Could you provide the updated diff?